arkane/encorr/data.py

```diff
@@ -28,391 +28,24 @@
-        # Table IX: Petersson GA (1998) J. of Chemical Physics, DOI: 10.1063/1.477794
-        'N-H': -0.42, 'C-N': -0.13, 'C#N': -0.89, 'C-F': 0.55, 'C-Cl': 1.29, 'S-H': 0.0, 'C-S': 0.43,
-        'O=S': -0.78,
-        'N=O': 1.11, 'N-N': -1.87, 'N=N': -1.58, 'N-O': 0.35,
-        # Table 2: Ashcraft R (2007) J. Phys. Chem. B; DOI: 10.1021/jp073539t
-        'N#N': -2.0, 'O=O': -0.2, 'H-H': 1.1,  # Unknown source
-    },
-
-    'cbs-qb3-paraskevas': {
-        # NOTE: The Paraskevas corrections are inaccurate for non-oxygenated hydrocarbons,
-        # and may do poorly in combination with the Petersson corrections
-        'C-C': -0.495, 'C-H': -0.045, 'C=C': -0.825, 'C-O': 0.378, 'C=O': 0.743, 'O-H': -0.423,
-        # Table2: Paraskevas, PD (2013). Chemistry-A European J., DOI: 10.1002/chem.201301381
-        'C#C': -0.64, 'C#N': -0.89, 'C-S': 0.43, 'O=S': -0.78, 'S-H': 0.0, 'C-N': -0.13, 'C-Cl': 1.29,
-        'C-F': 0.55,  # Table IX: Petersson GA (1998) J. of Chemical Physics, DOI: 10.1063/1.477794
-        'N-H': -0.42, 'N=O': 1.11, 'N-N': -1.87, 'N=N': -1.58, 'N-O': 0.35,
-        # Table 2: Ashcraft R (2007) J. Phys. Chem. B; DOI: 10.1021/jp073539t
-        'N#N': -2.0, 'O=O': -0.2, 'H-H': 1.1,  # Unknown source
-    },
-
-    # Identical corrections for 'b3lyp/cbsb7', 'b3lyp/6-311g(2d,d,p)', 'b3lyp/6-311+g(3df,2p)', 'b3lyp/6-31g(d,p)'
-    'b3lyp/cbsb7': {
-        'C-H': 0.25, 'C-C': -1.89, 'C=C': -0.40, 'C#C': -1.50,
-        'O-H': -1.09, 'C-O': -1.18, 'C=O': -0.01, 'N-H': 1.36, 'C-N': -0.44,
-        'C#N': 0.22, 'C-S': -2.35, 'O=S': -5.19, 'S-H': -0.52,
-    },
-    'b3lyp/6-311g(2d,d,p)': {
-        'C-H': 0.25, 'C-C': -1.89, 'C=C': -0.40, 'C#C': -1.50,
-        'O-H': -1.09, 'C-O': -1.18, 'C=O': -0.01, 'N-H': 1.36, 'C-N': -0.44,
-        'C#N': 0.22, 'C-S': -2.35, 'O=S': -5.19, 'S-H': -0.52,
-    },
-    'b3lyp/6-311+g(3df,2p)': {
-        'C-H': 0.25, 'C-C': -1.89, 'C=C': -0.40, 'C#C': -1.50,
-        'O-H': -1.09, 'C-O': -1.18, 'C=O': -0.01, 'N-H': 1.36, 'C-N': -0.44,
-        'C#N': 0.22, 'C-S': -2.35, 'O=S': -5.19, 'S-H': -0.52,
-    },
-    'b3lyp/6-31g(d,p)': {
-        'C-H': 0.25, 'C-C': -1.89, 'C=C': -0.40, 'C#C': -1.50,
-        'O-H': -1.09, 'C-O': -1.18, 'C=O': -0.01, 'N-H': 1.36, 'C-N': -0.44,
-        'C#N': 0.22, 'C-S': -2.35, 'O=S': -5.19, 'S-H': -0.52,
-    },
-
-}
-
-# Melius-type bond additivity correction parameters
-mbac = {}
+import importlib.util
+import os
+
+from rmgpy import settings
+
+quantum_corrections_path = os.path.join(settings['database.directory'], 'quantum_corrections', 'data.py')
+spec = importlib.util.spec_from_file_location("quantum_calculations", quantum_corrections_path)
+data = importlib.util.module_from_spec(spec)
+spec.loader.exec_module(data)
+
+# Assign the data here so that it can be imported
+atom_hf = data.atom_hf
+atom_thermal = data.atom_thermal
+SOC = data.SOC
+atom_energies = data.atom_energies
+pbac = data.pbac
+mbac = data.mbac
+freq_dict = data.freq_dict
```

Comment:
To do this flexibly, you could consider something like
```python
for k, v in vars(data).items():
    if not k.startswith('__'):
        vars()[k] = v
```
Do you think that sacrifices readability too much?

Reply:
Not at all, I like this a lot, thanks for the suggestion!